docs/content/v2.0/admin/yb-admin.md

```diff
@@ -534,18 +534,23 @@ yb-admin -master_addresses <master-addresses> modify_placement_info <placement_i
 ```
 
 - *master-addresses*: Comma-separated list of YB-Master hosts and ports. Default value is `localhost:7100`.
-- *placement_info*: Comma-delimited list of placements for *cloud*.*region*.*zone*. Default value is `cloud1.datacenter1.rack1`.
-- *replication_factor*: The number of replicas for each tablet.
+- *placement_info*: Comma-delimited list of placements for *cloud*.*region*.*zone*. Default value is `cloud1.datacenter1.rack1`. Optionally, a minimum replica count can be specified after a colon, default being 1. If a placement is specified multiple times, the total count from all mentions are taken.
+- *replication_factor*: The number of replicas for each tablet. This value should be more than the total of replica counts specified in *placement_info*.
 - *placement_id*: The identifier of the primary cluster, which can be any unique string. If not set, a randomly-generated ID will be used.
 
 **Example**
 
 ```sh
 $ ./bin/yb-admin --master_addresses $MASTER_RPC_ADDRS \
     modify_placement_info  \
-    aws.us-west.us-west-2a,aws.us-west.us-west-2b,aws.us-west.us-west-2c 3
+    aws.us-west.us-west-2a:2,aws.us-west.us-west-2b:3,aws.us-west.us-west-2c,aws.us-west.us-west-2b:2 9
```

Comment:
Let's not repeat a Placement block in the example since that would introduce more confusion. That is just for backward compatibility and we shouldn't encourage it to the user.

Reply:
Makes sense, will update